CMSgov/mint-app
# [EASI-2050] Fix NULLability of fields

# EASI-2050

## Changes and Description

- Fixed the NULLability of:
  - GraphQL fields
  - GraphQL return types
  - Go models
  - DB Schema

## How to test this change

`scripts/dev go:test` should test most resolvers don't break / try and set any NULL values where they shouldn't, but I'd recommend using the UI and testing for extra confidence!

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
